tests/integration/test_delegated_access.py

```diff
@@ -28,7 +28,7 @@ def test_default_integration():
     ds.delete()
 
 
-@pytest.mark.skipif(os.environ.get("DA_GCP_LABELBOX_API_KEY") is None,
+@pytest.mark.skipif(os.environ.get("DA_GCP_LABELBOX_API_KEY") == "",
```

Comment:
This should be:
```@pytest.mark.skipif(os.environ.get("DA_GCP_LABELBOX_API_KEY"))```
That way this test doesn't run if either the env var is not set or if it is set to an empty string.

Reply:
`skipif` only accepts boolean as the condition, so I am doing `not os.environ.get("DA_GCP_LABELBOX_API_KEY")` instead, which should evaluate to false for empty string and None

Reply:
Good call nonetheless!